src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -291,6 +291,11 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 let offset_ptr = ptr.ptr_wrapping_signed_offset(offset_bytes, self);
                 self.write_scalar(offset_ptr, dest)?;
             }
+            sym::ptr_guaranteed_eq | sym::ptr_guaranteed_ne => {
+                // FIXME: return `true` for at least some comparisons where we know equality and
+                // inequality have meaning.
```

Comment:
```suggestion
                // FIXME: return `true` for at least some comparisons where we can determine (in)equality
                // at compile time.
```
"have meaning" doesn't really have much meaning here IMO. ;)

Reply:
I rewrote the FIXME